support/sdk-test/entrypoint.sh

```diff
@@ -14,6 +14,12 @@ dnf install -y --setopt=tsflags=noscripts avocado-sdk-toolchain
 mkdir -p /opt/avocado/sdk/avocado-qemux86-64/0.1.0/sysroots/core2-64-avocado-linux/var
 dnf -y --setopt=tsflags=noscripts --installroot /opt/avocado/sdk/avocado-qemux86-64/0.1.0/sysroots/core2-64-avocado-linux/ install packagegroup-core-standalone-sdk-target
 
```

Comment:
Lets add a comment here that this can be removed once the toolchain can install these files.

Reply:
done